javascript/javascript1/week2/hw2.js

```diff
@@ -0,0 +1,90 @@
+// Flight booking fullname function
+
+function getFullname(firstname, surname){
+    /*firstname = 'Benjamin';
+    surname = 'Hughes';*/
+    return firstname + ' ' + surname;
+}
+//console.log(getFullname());
+var fullname1 = getFullname('John' , 'Doe');
+var fullname2 = getFullname('Lina' , 'Doe');
+console.log(fullname1);
+console.log(fullname2);
+
+
+//Formal fullname
+
+function getFullname(firstname, surname, useFormalName){
+   
+return     useFormalName=== true ? 'lord'+ ' ' +firstname +' '+ surname
+         : useFormalName=== false ? firstname +' ' + surname
+         :  undefined ;
+  }
+  
+      console.log(getFullname('john', 'doe',true));
+
+//Eventday
+const date = new Date();
+const day = date.getDay();
+const weekday = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+function getEventWeekday(daysToEvent){
+  const eventDay = (daysToEvent + day) % 7;
+  return eventDay;
+}
+ const dayOfEvent = getEventWeekday(2);
+ console.log(weekday[dayOfEvent]);
+  
+ //Weather wear
+  
+  function weatherForeCast(temperature){
+    if(temperature >= 15){
+      return 'Shorts' + ' ' + ' T-shirts';
+    }else if(temperature < 15 ){
+      return  'full covered clothes';
+    }else if(temperature <= 0){
+      return  'winter clothes '
+    }else {
+      return 'rainy clothes';
+    }
+  }
+  var clothesToWear = weatherForeCast();
+  console.log(clothesToWear);
+  // student names
+  const class07Students = [];
+  function addStudentToClass(studentName) {
+    // You write code here
+  if(class07Students.length < 6)
```

Comment:
Give the conditions based on priority and more readable code.The conditions & returns are repeated.